packages/compiler/src/core/helpers/type-name-utils.ts

```diff
@@ -182,7 +182,16 @@ function getInterfaceName(iface: Interface, options: TypeNameOptions | undefined
 }
 
 function getOperationName(op: Operation, options: TypeNameOptions | undefined) {
-  return `${getNamespacePrefix(op.namespace, options)}${getIdentifierName(op.name, options)}`;
+  let opName = getIdentifierName(op.name, options);
+  if (op.node.templateParameters.length > 0) {
+    // template
+    const params = op.node.templateParameters.map((t) => getIdentifierName(t.id.sv, options));
+    opName += `<${params.join(", ")}>`;
+  }
```

Comment:
@timotheeguerin , is there any case that template instantiated is true for operation's template arguments? I can only find it's true when the parent interface is used by alias, so skipped that part here for operation.

And about the template instantiated, is it only true when it's used by alias? I found for some statement like "op Op1 is Op2", the template instantiated is still false for Op2. is that expected?